README.md

```
@@ -73,27 +98,44 @@ To switch back to Cursor's API:
+|-------------|------------------|
+| `claude-4.5-opus` | `claude-opus-4.5` |
+| `claude-4.5-sonnet` | `claude-sonnet-4.5` |
+| `gpt-4-turbo` | `gpt-4-0125-preview` |
+| `gemini-3-pro` | `gemini-3-pro-preview` |
+
+You can use either format - the proxy handles the translation.
 
-### Linting:
+## 🛠️ Development
+
+### Available Scripts:
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)